recipes/autoconf/all/conanfile.py

```diff
@@ -68,64 +73,66 @@ def generate(self):
                 tc.configure_args.append(f"--host={host}")
 
         env = tc.environment()
-        env.define("INSTALL", unix_path(self,  path.join(self.source_folder, 'build-aux', 'install-sh')))
+        env.define_path("INSTALL", unix_path(self, os.path.join(self.source_folder, "build-aux", "install-sh")))
         tc.generate(env)
 
-        deps = AutotoolsDeps(self)
-        deps.generate()
-
-        ms = VirtualBuildEnv(self)
-        ms.generate(scope="build")
-
-    def build(self):
+    def _patch_sources(self):
         apply_conandata_patches(self)
-        replace_in_file(self, path.join(self.source_folder, "Makefile.in"), "M4 = /usr/bin/env m4", "#M4 = /usr/bin/env m4")
+        replace_in_file(self, os.path.join(self.source_folder, "Makefile.in"),
+                        "M4 = /usr/bin/env m4", "#M4 = /usr/bin/env m4")
 
+    def build(self):
+        self._patch_sources()
         autotools = Autotools(self)
         autotools.configure()
         autotools.make()
 
     def package(self):
         autotools = Autotools(self)
-        autotools.install(args=[f"DESTDIR={unix_path(self, self.package_folder)}"])  # Need to specify the `DESTDIR` as a Unix path, aware of the subsystem
+        # TODO: can be replaced by autotools.install() if required_conan_version = ">=1.54.0"
+        autotools.install(args=[f"DESTDIR={unix_path(self, self.package_folder)}"])
```

Comment:
Maybe add a FIXME comment once CCI uses the Conan version with your pr https://github.com/conan-io/conan/pull/12193

Reply:
it's fixed in 1.54.0 so for me the TODO comment has already everything